scripts/resources/tachidesk-browser-launcher-aur.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec /usr/bin/java -jar /usr/share/java/tachidesk/tachidesk.jar
```

Comment:
rename these new launchers to `xx-launcher-debian.sh` or `xx-launcher-standalone.sh`